templates/zerver/hello.html

```diff
@@ -40,7 +40,7 @@ <h1>The world&rsquo;s most productive team chat</h1>
             <div class="tour">
                 <div id="tour-carousel" class="carousel slide carousel-fade">
                     <!-- Carousel items -->
-                    <div class="carousel-inner">
+                    <a><div class="carousel-inner">
```

Comment:
I would rather add
`.tour .carousel-inner .item-container:hover {
    cursor: pointer;
}`
to the CSS than adding an `<a>` here.